recipes/la_forge/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "la_forge" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0b457003af2bcfe49b3320708520eb9e93a7b882f6b318b388f3107b9f630eee
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - numpy >=1.16
+    - scipy >=1.0.0
+    - matplotlib-base
+    - astropy >=3.0
+    - corner
+    - six
+    - h5py >=3.4.0
+
+test:
+  imports:
+    - la_forge
+    - la_forge.diagnostics
+    - la_forge.core
+    - la_forge.gp
+    - la_forge.rednoise
+    - la_forge.slices
+    - la_forge.utils
+
+about:
+  home: https://github.com/nanograv/la_forge
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Pulsar Timing Array Bayesian Data Visualization'
+  description: |
+    Python package for conveniently plotting results from pulsar timing array bayesian analyses. Many of the functions are best used with enterprise outputs.
+  doc_url: https://la-forge.readthedocs.io/
+  dev_url: https://github.com/hazboun6/la_forge
```

```suggestion
  dev_url: https://github.com/nanograv/la_forge
```

Just to make it consistent.